.buildkite/pipeline-for-tags.yml

```diff
@@ -19,8 +19,8 @@ steps:
      queue: "docker"
    commands:
    - eval "$SET_VERSION"
-   - nix develop .#docker-tezos-packages -c ./docker/docker-tezos-packages.py --os ubuntu --type source
-   - nix develop .#docker-tezos-packages -c ./docker/docker-tezos-packages.py --os fedora --type source
+   - nix develop .#docker-tezos-packages -c ./docker/docker-tezos-packages.py --os ubuntu --type source -s "Serokell <tezos-packaging@serokell.io>"
```

Comment:
I quickly tried this out [here](https://buildkite.com/serokell/tezos-packaging/builds/3385#0185c9f2-652c-4488-a1b2-d8b7032bc676) and it ends in a failure.

AFAICT this is caused by the fact that `debsign` is not made available by `nix develop .#docker-tezos-packages`.
I suspect that the same is true for `rpmsign` as well.

Reply:
resolved, everything squashed
`docker-tezos-packages` now points to another shell, I deleted previous one